src/lib/staged_ledger/staged_ledger.ml

```diff
@@ -555,6 +563,46 @@ end = struct
     ; user_commands_count= List.length user_commands
     ; coinbase_parts_count= List.length coinbase }
 
+  (*The total fee excess caused by any diff should be zero. In the case where the slots are split into two partitions, total fee excess of the transactions to be enqueued on each of the partitions should be zero respectively *)
```

Comment:
could this comment be broken into multiple lines ... it gets truncated in Github, and might do so in an editor.

Reply:
Use `(**` to open doccomment